app/views/content/train-to-be-a-teacher/subject-knowledge-enhancement.md

```diff
@@ -23,15 +23,29 @@ SKE courses can help when you have:
 - a different but related degree to the subject you want to teach
 - an A level but not a degree in the subject
 - an unrelated degree but relevant professional experience
-- a languages degree but need to improve your knowledge of a language you’ll teach
+- studied for a languages degree, but need to develop a second language to an acceptable level for teaching in schools 
 - not used your subject knowledge for a while
 
-When you apply for teacher training, the school or university will help you establish whether you need to improve your subject knowledge.
+When you apply for teacher training, the school or university will help you establish whether you need to improve your subject knowledge. They may decide to make completing a SKE course a condition of your offer to train with them.
```

Comment:
@EllieNodder Should we say 'school or university' here or should we say 'teacher training provider' or something similar?

Reply:
yep this should probably be 'your teacher training provider'